osdep/drm-hdr-metadata.h

```diff
@@ -0,0 +1,89 @@
+#pragma once
```

Comment:
I think you can get away without this header all together. All the required code is already properly guarded, albeit the two defines used make it a bit confusing.